src/index.js

```diff
@@ -22,5 +21,4 @@ export default base.merge(
   eslint,
   flowStatus,
   production,
-  browserSync,
```

Comment:
@alebiavati is it not worth just removing the extension from the `base` export, but keeping it in the project? Although we're not using it for our project, it may be useful to provide the extension in case someone else wants to use it down the road? just a thought